packages/react-components/react-popover/src/components/Popover/Popover.types.ts

```diff
@@ -87,6 +87,13 @@ export type PopoverProps = Pick<PortalProps, 'mountNode'> & {
    */
   openOnHover?: boolean;
 
+  /**
+   * Flag to close the Popover when an iframe outside a PopoverSurface is focused
+   *
+   * @default false
```

Comment:
nit: update the PR desription, the default value of this prop is `false`

Reply:
Updated the comment, it's actually `true` by default:

<img width="623" alt="image" src="https://github.com/microsoft/fluentui/assets/14183168/1fdb4b0b-5960-4435-9c06-4d3506352bb9">